translations/translations-workflow.md

```diff
@@ -0,0 +1,99 @@
+# Translations workflow
+
+We're using weblate as a web service to help the translation process, through
+the hosted service of weblate: https://hosted.weblate.org/projects/rero_plus
+
+For our needs, on weblate, we have one project (name RERO +, with `rero_plus`
+as slug), with many *components*:
+
+- `ng-core`.
+- `rero-ils`.
+- `rero-ils-public-search` (from the GitHub `rero-ils-ui`).
+- `rero-ils-amdin` (from the GitHub `rero-ils-ui`).
+- `sonar`.
+- `sonar-ui`.
+
+## The *very* first time
+
+When every existing PR that contain modifications on the translated files are
+merged into the `dev` branch of the corresponding GitHub project:
+
+1. Verify that the weblate component is up to date with the GitHub project. To
+   do that you can either:
+    - Got to
+      https://hosted.weblate.org/projects/rero_plus/<component>/#repository and
+      check the repository status.
+    - Or clone the weblate `git` repository and inspect the `git` logs (`git
+      clone
+      'https://hosted.weblate.org/git/rero_plus/<component>/')`.
+    If it not, check [the workflow][3] below, to know how to update it.
+2. Verify that on the GitHub repository settings, the webhook is active.
+3. Unlock the tranlsations (you can do that on the web interface, or through
+   the [`wlc` client][4]).
+4. Inform the translators that weblate is ready.
+
+## The regular workflow
+
+For the sychronisation workflow, it's semi automatic:
+
+- A GitHub application is installed in the RERO GitHub organization.
+- In each repository, a web hook is configured, as [explained in the weblate
+  documentation][1].
+
+This ensure two things:
+
+1. Web changes are merged into the `dev` branch of our repository, the weblate
+   service is informed through the webhook and pulls changes to the weblate
+   `git` repository. It's done with a `git rebase` command, as it can be
+   configured in the component settings [component/settings/version control].
+1. As translations are being made through the weblate interface, commits are
+   being made. 24 hours after a commit, a PR is created on the corresponding
```

Comment:
I would add a precision on the fact that 24 hours is configurable in Weblate administration panel (or somewhere else)